Sources/SpeziViews/Environment/ProcessingDebounceDuration.swift

```diff
@@ -0,0 +1,29 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
```

Comment:
Very small thing, but I generally add two lines between the import statement and the beginning of the code. , Also applies to other parts of the PR
```suggestion


```